packages/headless/src/database/mongodb/order/index.ts

```diff
@@ -354,4 +355,32 @@ export class OrderDatabase implements IOrderDatabase {
       return false;
     }
   }
+
+  async createReply(request: ICreateReply): Promise<IReviewReplyResponse | null>{
+    const { reviewId } = request;
+    delete request.reviewId;
+
+    try{
+      const reply =  await ReviewModel.findOneAndUpdate(
+        { id: reviewId },
+        { reply: request},
+        { new: true }
+      ).select('-_id').lean().exec();
+
+      if(reply) return null;
```

Comment:
didn't understand the logic here. can you clarify?

Reply:
if it is to just check `whether a review already exists or not` then use another function to get that info